src/mongo/delegates/Tables.class.php

```diff
@@ -564,17 +569,17 @@ public function generateTableRows($tableType, $resource = null, $context = null,
                         _ID_CONTEXT => $doc['_id'][_ID_CONTEXT],
                         _ID_TYPE=>$tableSpec['_id']
                     ],
-                    \_CREATED_TS => \Tripod\Mongo\DateUtil::getMongoDate()
```

Comment:
This is moved to before we fetch the source documents to prevent invalidation timing issues.